Client/src/Pages/Monitors/Home/actionsMenu.jsx

```diff
@@ -44,6 +47,28 @@ const ActionsMenu = ({ monitor, isAdmin, updateCallback }) => {
     }
   };
 
+  const handlePause = async (event) => {
+    try {
+      const action = await dispatch(
+        pauseUptimeMonitor({ authToken, monitorId: monitor._id })
+      );
+      if (pauseUptimeMonitor.fulfilled.match(action)) {
+        updateCallback();
+        //window.location.reload();
+        if (action.payload?.isActive === false){
+          createToast({ body: "Monitor paused successfully." });
+        }else{
+          createToast({ body: "Monitor resumed successfully." });
+        }
+      } else {
+        throw new Error(action.error.message);
```

Comment:
It's a good idea to safely access nested properties:

```
  throw new Error(action?.error?.message ?? "Failed to pause monitor.");
```

`action` isn't guaranteed to have `error` defined, so we shoudl fail gracefully if it isn't